internal/mapobj/map.gen.go

```diff
@@ -44,8 +44,7 @@ func (m {{ .Type.Name }}) Validate() error {
     for k := range m {
     	keys = append(keys, k)
     }
-    strings.Sort(keys)
-    for k := range keys {
+    for _, k := range keys {
```

Comment:
Is there a reason to remove the sort?

Reply:
I didn't find a way to import `sort`/`strings` package inside generated file.
And seems like this is a bit useless here.

Reply:
If you're just putting things in a map, go maps are unsorted/random, and thus it's actually a perf bug (maybe)